### ISSUE TITLE
4156091B - Tony Hawk's Pro Skater 5

### ISSUE THREAD

**Issue by spectrumm** [xenia-project/game-compatibility#751]
[Marketplace](http://marketplace.xbox.com/...)

Tested on (https://ci.appveyor.com/project/benvanik/xenia/build/1.0.679-master)

# Issues:
Loads till first splash logo and guest proceeds to crash

# Log:
[Log file](https://gist.github.com/spectrumm/0fdefbfa503ed3a38503299471e30f7f)

# Screenshot(s):
![](https://cdn.discordapp.com/attachments/248756880388653058/303997795453632532/unknown.png)
![](https://cdn.discordapp.com/attachments/248756880388653058/303997857441120256/unknown.png)

# Labels:
state-crash

**Comment by Sleepy93** [comment]
well :) 

There is an unimplemented feature :) 

> w> 00000028 ERROR: fn export referenced GetProcAddressByOrdinal(0AFF(XamPartyGetUserList)) is not implemented
> w> 00000028 ERROR: XexGetProcedureAddress ordinal not found!
> i> 00000028 RtlNtStatusToDosError(C0000263)
> i> 00000028 RtlNtStatusToDosError => 7F
> w> 00000004 GPU: Write to unknown register (0E00 = 00000001)

And same with a lot of games GPU write to unknown register :) 

# Labels:
- state-intro
- state-crash 
- kernel-unimplemented-feature

what do you think?

**Comment by spectrumm** [comment]
@Sleepy93 I agree, didnt have enough time to properly look through the log :p 

Another question, do you have NVIDIA or AMD GPU?

**Comment by Sleepy93** [comment]
@spectrumm I have MSI GTX750 Ti so nvidia :)

**Comment by DarkKnight64** [comment]
is it even worth trying to emu this game lol

**Comment by Sleepy93** [comment]
# Update
Goes into menu but hangs on loading screen when I start a new game.

# Tested 
On (https://github.com/xenia-project/xenia/commit/2cebd3cabecfeedc643b4116debcb3fa6c5cc33e)

# Issues

# Undefined

# Unimplemented
- XamPartyGetUserList
- XamPartySendGameInvites
- XamPartySetCustomData
- XamPartyGetBandwidth
- XamShowPartyUI
- XamShowCommunitySessionsUI
- XamGetServiceEndpoint

# GPU
!> F8000028 GPU: Read from unknown register (044B)
Writes to unk. register
- (0081 = 80010000)
- (0082 = 00000000)
- (1E4E = 00000000)

# Log file
[xenia.log](https://github.com/xenia-project/game-compatibility/files/5241746/xenia.log)

# Screens

![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_51_03](https://user-images.githubusercontent.com/20132031/93527039-9a463a80-f938-11ea-80df-45bff06845e6.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_51_08](https://user-images.githubusercontent.com/20132031/93527047-9ca89480-f938-11ea-9403-7ae4f8ea0da2.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_51_13](https://user-images.githubusercontent.com/20132031/93527048-9ca89480-f938-11ea-9e28-548abc130d2f.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_51_16](https://user-images.githubusercontent.com/20132031/93527049-9d412b00-f938-11ea-905d-ef370727a091.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_51_19](https://user-images.githubusercontent.com/20132031/93527061-9f0aee80-f938-11ea-8996-3f45c03938a6.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_51_24](https://user-images.githubusercontent.com/20132031/93527070-a16d4880-f938-11ea-8b93-b535bcacd9f9.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_51_26](https://user-images.githubusercontent.com/20132031/93527080-a3cfa280-f938-11ea-8074-66bf401e93cb.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_51_36](https://user-images.githubusercontent.com/20132031/93527083-a4683900-f938-11ea-878a-ab42b605a8c1.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_51_40](https://user-images.githubusercontent.com/20132031/93527084-a4683900-f938-11ea-84d2-d4dd3150655a.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_51_44](https://user-images.githubusercontent.com/20132031/93527085-a500cf80-f938-11ea-9432-879f6114495a.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_51_48](https://user-images.githubusercontent.com/20132031/93527086-a500cf80-f938-11ea-88fb-0cc23f05bbee.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_51_53](https://user-images.githubusercontent.com/20132031/93527088-a5996600-f938-11ea-9f4a-6e362dbe752d.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_51_58](https://user-images.githubusercontent.com/20132031/93527094-a7fbc000-f938-11ea-8d27-24d5b8702d36.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_52_06](https://user-images.githubusercontent.com/20132031/93527099-a9c58380-f938-11ea-8d62-47c0c550af99.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_52_10](https://user-images.githubusercontent.com/20132031/93527104-aaf6b080-f938-11ea-9f6f-84b660cb6775.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_52_36](https://user-images.githubusercontent.com/20132031/93527107-ac27dd80-f938-11ea-949b-9712748c21d8.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_52_43](https://user-images.githubusercontent.com/20132031/93527110-acc07400-f938-11ea-8098-0f9615f74d25.png)
![xenia (detached_2cebd3ca_Sep 15 2020) _  4156091B  Pro Skater™ 5 _Direct3D 12_ 17_09_2020 22_52_49](https://user-images.githubusercontent.com/20132031/93527112-adf1a100-f938-11ea-9fc2-d4c5d86b7b5c.png)

# Labels
- state-menus
- tech-engine-unreal
- kernel-unimplemented-feature

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/e7cd2ffffa418002ea5e526a3edba734d5f24990

Started a new game and sk8ed around hub area. 62FPS with vsync off and shaders cached.

![thp (4)](https://user-images.githubusercontent.com/31154606/103294572-bdf93200-49b7-11eb-9c6a-33eb7fbb09a1.png)

<details>
  <summary>More Screenshots</summary>

![thp (1)](https://user-images.githubusercontent.com/31154606/103294576-bf2a5f00-49b7-11eb-8554-a21c83bd346c.png)
![thp (2)](https://user-images.githubusercontent.com/31154606/103294578-bf2a5f00-49b7-11eb-90b0-ac1805d1797f.png)
![thp (3)](https://user-images.githubusercontent.com/31154606/103294579-bfc2f580-49b7-11eb-980e-d1ad0452699e.png)
![thp (5)](https://user-images.githubusercontent.com/31154606/103294574-be91c880-49b7-11eb-8855-f3c9ba331220.png)
![thp (6)](https://user-images.githubusercontent.com/31154606/103294575-be91c880-49b7-11eb-857a-eccd80b49fb8.png)

</details>

state-gameplay | tech-middleware-wwise

**Comment by homeless-field** [comment]
Tested on https://github.com/xenia-project/xenia/commit/dbd518f13

Tested the first park, some challenges, character creation, and park creation. Only minor issues

Issues:
* Texture flickering on graffiti and such

Log:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/7465919/xenia.zip)

Screenshot:
![Screenshot](https://user-images.githubusercontent.com/59347006/140022999-5e99a36c-cbf8-4e66-b332-f767aab0a72f.png)

Labels:
state-playable
